### ISSUE TITLE
IxGroup does not show expand icon

### ISSUE THREAD

**Issue by tomas-reznak** [siemens/ix#1206]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [X] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### What happened?

When I try exactly the same code as provided in [IxGroup usage](https://ix.siemens.io/docs/controls/group#usage) (React section), the icon for expanding the group items is not rendered initially.

In some cases (like after adding "collapsed={false}" prop to the component) it is displayed, but basically, the behavior is strange.

Current wrong behavior:

![image](https://github.com/siemens/ix/assets/99844553/6fe9f097-b2e3-4180-a824-bcdfd87b7bbd)

Correct behavior:

![image](https://github.com/siemens/ix/assets/99844553/ef5f7470-d189-4793-beba-0bc079301671)

Details:

When I try browser dev tools, expand icon has "hidden" class, which causes (probably) the problem:
 
![image](https://github.com/siemens/ix/assets/99844553/30fc89c0-405e-4376-ac9d-56c4f83c864c)

### What type of frontend framework are you seeing the problem on?

React

### Which version of iX do you use?

v2.1.2

### Code to produce this issue.

```shell
<IxGroup header="Group header" subHeader="Group subheader">
	<IxGroupItem text="Group item 1" />
	<IxGroupItem text="Group item 2" />
	<IxGroupItem text="Group item 3" />
</IxGroup>
```

**Comment by danielleroux** [comment]
@tomas-reznak can try to update to latest version or define the ix-icons in your main.ts before starting the react app

e.g:
```
import { defineCustomElements } from '@siemens/ix-icons/loader';
await defineCustomElements();
```

**Comment by tomas-reznak** [comment]
I tried to update packages:
![image](https://github.com/siemens/ix/assets/99844553/a837beaf-c555-4308-ba3f-2cf8174de962)

and also changed the loader for custom elements (we used to import `defineCustomElement` from `@siemens/ix-brand-theme/loader`, not from `ix-icons`), but didn't help.

I thing the problem is not caused by missing icon, because this can be displayed in some cases - e.g. if I add ` suppressHeaderSelection={true}` to the `IxGroup` element and then save the file with code, it somehow works. But after refreshing the page, the expend icon is not shown again and after clicking on the header it appears. Weird...

I tried to record the behavior we actually have:

![ix-group-carret-not-shown2](https://github.com/siemens/ix/assets/99844553/7e05d7b1-7694-4302-afd9-f4aceced1add)
